### ISSUE TITLE
Invoke external download manager

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#4454]
Using an external downloader for downloads is something that came up a couple of times in the past. It probably won't work for all cases (as it needs to do a second request, and doesn't have cookies), but I suppose it's still worthwhile.

**Comment by user202729** [comment]
I don't think this is possible for POST request, as the QWebEngineDownloadItem only allows getting the URL.

**Comment by The-Compiler** [comment]
Not that I'm aware of. If there were any, you'd likely see them here.

**Comment by Cybolic** [comment]
I thought this would be possible using the `interceptor` API, but, unless I'm mistaken, downloads don't pass through there, right?

**Comment by wgmayer0** [comment]
I ended up using a hint and assigning my downloader program to that via a
userscript
